aptos-move/framework/aptos-token/sources/token.move

```diff
@@ -376,6 +376,65 @@ module aptos_token::token {
         *opt_in_flag = opt_in;
     }
 
+    public entry fun mutate_collection_description(
+        account: &signer,
+        creator: address,
+        collection_name: String,
+        description: String
+    ) acquires Collections {
+        assert!(signer::address_of(account) == creator, ENO_MUTATE_CAPABILITY);
+        assert!(exists<Collections>(creator), ECOLLECTIONS_NOT_PUBLISHED);
```

Comment:
Wrap error code

Reply:
will wrap it into error::permission_denied